frontend/public/components/resource-quota.jsx

```diff
@@ -1,38 +1,30 @@
+import * as _ from 'lodash-es';
 import * as React from 'react';
 
-import { ColHead, DetailsPage, List, ListHeader, ListPage } from './factory';
+import { ColHead, DetailsPage, List, ListHeader, MultiListPage } from './factory';
 import { Cog, SectionHeading, navFactory, ResourceCog, ResourceLink, ResourceSummary } from './utils';
-import { registerTemplate } from '../yaml-templates';
-
-registerTemplate('v1.ResourceQuota', `apiVersion: v1
-kind: ResourceQuota
-metadata:
-  name: example
-spec:
-  hard:
-    pods: "4"
-    requests.cpu: "1"
-    requests.memory: 1Gi
-    limits.cpu: "2"
-    limits.memory: 2Gi
-`);
-
 
 const menuActions = [Cog.factory.ModifyLabels, Cog.factory.ModifyAnnotations, Cog.factory.Edit, Cog.factory.Delete];
 
+export const QuotaName = ({quota: quota}) => {
+  const kind = quota.metadata.namespace ? 'ResourceQuota' : 'quota.openshift.io:v1:ClusterResourceQuota';
```

Comment:
You should be able to use `referenceFor` (but it also shouldn't be needed if you register the ClusterResourceQuota model)

https://github.com/openshift/console/blob/2c7a4b0b31a9046ad03863382a968e386273b6db/frontend/public/module/k8s/k8s.ts#L18